Sources/Sentry/SentryClient.m

```diff
@@ -63,6 +63,16 @@ - (_Nullable instancetype)initWithOptions:(SentryOptions *)options
         self.threadInspector =
             [[SentryThreadInspector alloc] initWithStacktraceBuilder:stacktraceBuilder
                                             andMachineContextWrapper:machineContextWrapper];
+
+        NSError *error = nil;
+        self.fileManager = [[SentryFileManager alloc] initWithDsn:self.options.parsedDsn
+                                                 didFailWithError:&error];
+        if (nil != error) {
+            [SentryLog logWithMessage:error.localizedDescription andLevel:kSentryLogLevelError];
```

Comment:
H: Should we continue here?

We'd be passing the null ref to the transport below.
Since we need the fileManager to pretty much everything, it's likely best to just set the SDK to no-op mode (disabled).